### PR TITLE
Release branch 1.6.0

### PR DESCRIPTION
**Pull request by artnowo-alle**

### CODE REVIEW

File: bigflow/_version.py [hunk 1]
```diff
@@ -1,2 +1 @@
-__version__ = '1.5.5dev2'
-
+__version__ = '1.7.0.dev1'
```
Comment (grzegorzwitkowski) [inline bigflow/_version.py:1]: hmm, doesn't match with the PR title `Release branch 1.6.0`

Comment (artnowo-alle) [reply bigflow/_version.py:1]: Yeah, but 1.6.0 is already released (I just tagged commit [e6b2c32](https://github.com/allegro/bigflow/pull/360/commits/e6b2c324298a5e0188a1fd1d335a2cecb3de170f)). If I would leave 1.6.0 here it'd try to release it again. What I want to get is to release 1.7.0.dev1 once it's merged to master.

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -5,16 +5,23 @@ on:
     branches: [ master ]
     types:
       - completed
+  workflow_dispatch:
+    inputs:
+      ref:
```
Comment (agnieszkarybak) [inline .github/workflows/release.yml:10]: Do we need this input? Do we need to run the workflow from one branch's HEAD but release the version from another commit?

**Comment by artnowo-alle** [comment]
Actual change released in 1.6.0 is in PR #357

File: .github/workflows/release.yml (hunk 2 above)
Comment (artnowo-alle) [reply .github/workflows/release.yml:10]: I think we might want to do this. I'm fine with dropping it now and adding later if needed, though

**Review by grzegorzwitkowski — APPROVED** [review]

**Review by agnieszkarybak — APPROVED** [review]

File: .github/workflows/release.yml (hunk 2 above)
Comment (artnowo-alle) [reply .github/workflows/release.yml]: Agreed with Agnieszka to remove it. One can install a requirement from a specific commit, branch, or tag instead.

**Pull request merged**